spec/models/collection_branding_info_spec.rb

```diff
@@ -40,6 +40,15 @@
       expect(logo_info.alt_text).to eq("This is the logo")
       expect(logo_info.target_url).to eq("http://logo.com")
     end
+
+    it "saves the logo info, but don't upload the log file" do
+      logo_info.save(file.path, false)
+
+      expect(storage_adapter).not_to receive(:upload)
```

Comment:
```suggestion
      expect(storage_adapter).not_to receive(:upload)

      logo_info.save(file.path, false)
```

The check to see if `storage_adapter.upload` needs to be checked before the call to `logo_info.save`; otherwise, the expect will always pass.